apps/src/studio/rocketHeightLogic.js

```diff
@@ -31,6 +31,7 @@ RocketHeightLogic.prototype.onTick = function () {
     this.seconds = 0;
     this.rocket = this.studio_.sprite[this.rocketIndex];
     this.height = 0;
+    this.studio_.setBackground({value: 'space'});
```

Comment:
Not strictly necessary. Right now this is something we do with hidden non-functional blocks (i.e. when run -> setBackground). Seems cleaner to have that logic here, so that if we create a new big game level we don't have to worry about remembering to create those hidden blocks.